packages/icons/src/order-first.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from "react";
+import { IconProps } from "./types";
+
+export const OrderFirstIcon = React.forwardRef<SVGSVGElement, IconProps>(
+  ({ color = "currentColor", ...props }, forwardedRef) => {
+    return (
+      <svg
+        xmlns="http://www.w3.org/2000/svg"
+        fill="none"
+        viewBox="0 0 16 16"
+        width="16"
+        height="16"
+        {...props}
+        ref={forwardedRef}
+      >
+        <mask id="a" fill="#fff">
+          <rect width="4" height="8" x="10.9" y="4" rx="1" />
+        </mask>
+        <rect
+          width="4"
+          height="8"
+          x="10.9"
+          y="4"
+          stroke={color}
+          strokeWidth="2.2"
+          mask="url(#a)"
+          rx="1"
+        />
+        <mask id="b" fill="#fff">
```

Comment:
is svg becoming part of the global document when rendered directly as a component? I wonder if that id is only visible inside the SVG itself or it just became a global id

Reply:
I understand its from figma, my question is if that's ok or is it a document-global id

Reply:
We can add prefixes for ids in svg with svgo plugin